### PR TITLE
Fixes for 2.3 branch

### PR DESCRIPTION
**Pull request by ifdattic**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.3 |
| Fixed tickets |  |

### CODE REVIEW

File: reference/forms/types/checkbox.rst [hunk 1]
```diff
@@ -5,8 +5,8 @@ checkbox Field Type
 ===================
 
 Creates a single input checkbox. This should always be used for a field that
-has a Boolean value: if the box is checked, the field will be set to true,
-if the box is unchecked, the value will be set to false.
+has a ``Boolean`` value: if the box is checked, the field will be set to ``true``,
```
Comment (WouterJ) [inline reference/forms/types/checkbox.rst]: boolean is just a name (I would love to make it lowercase).

File: reference/forms/types/choice.rst [hunk 2]
```diff
@@ -86,11 +86,11 @@ choices
 **type**: ``array`` **default**: ``array()``
 
 This is the most basic way to specify the choices that should be used
-by this field. The ``choices`` option is an array, where the array key
+by this field. The ``choices`` option is an ``array``, where the array key
```
Comment (WouterJ) [inline reference/forms/types/choice.rst]: same here

File: reference/forms/types/choice.rst [hunk 3]
```diff
@@ -111,7 +111,7 @@ The ``choice_list`` option must be an instance of the ``ChoiceListInterface``.
 For more advanced cases, a custom class that implements the interface
 can be created to supply the choices.
 
-With this option you can also allow float values to be selected as data.
+With this option you can also allow ``float`` values to be selected as data.
```
Comment (WouterJ) [inline reference/forms/types/choice.rst]: an here ...

File: reference/forms/types/entity.rst [hunk 4]
```diff
@@ -140,7 +140,7 @@ property
 
 This is the property that should be used for displaying the entities
 as text in the HTML element. If left blank, the entity object will be
-cast into a string and so must have a ``__toString()`` method.
+cast into a ``string`` and so must have a ``__toString()`` method.
```
Comment (WouterJ) [inline reference/forms/types/entity.rst]: an all other places where you used types (I'll stop commenting on them now)

File: reference/forms/types/checkbox.rst (hunk 1 above)
Comment (xabbuh) [inline reference/forms/types/checkbox.rst]: see #4671 :)

Comment (stof) [inline reference/forms/types/checkbox.rst]: it should not be in backticks here. It is about the English adjective in this sentence

**Comment by WouterJ** [comment]
Can you please update this PR, @ifdattic ?

**Comment by ifdattic** [comment]
@WouterJ I created a replacement PR https://github.com/symfony/symfony-docs/pull/5116

**Comment by xabbuh** [comment]
Closing as #5116 was merged.

**Pull request closed without merging**